core/src/components/button/button.tsx

```diff
@@ -313,7 +313,9 @@ export class Button implements ComponentInterface, AnchorInterface, ButtonInterf
           <span class="button-inner">
             <slot name="icon-only"></slot>
             <slot name="start"></slot>
-            <slot></slot>
+            <span class="button-text">
```

Comment:
I'm open to suggestions on the naming of this class. Since it can technically hold things other than text, maybe it should be something more vague.

Reply:
What about `button-content`?